src/main/java/core/basesyntax/IFigure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface IFigure {
```

Comment:
what does this name mean? Let’s make this name more informative. For example "AreaCalculator"